olp-cpp-sdk-dataservice-read/src/generated/model/SubscribeResponse.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#pragma once
+
+#include <string>
+
+#include <boost/optional.hpp>
+
+namespace olp {
+namespace dataservice {
+namespace read {
+namespace model {
+
+class SubscribeResponse final {
+ public:
+  const boost::optional<std::string>& GetNodeBaseURL() const {
+    return node_base_url_;
+  }
+  void SetNodeBaseURL(std::string value) { node_base_url_ = std::move(value); }
+
+  const boost::optional<std::string>& GetSubscriptionId() const {
+    return subscription_id_;
+  }
+  void SetSubscriptionId(std::string value) {
+    subscription_id_ = std::move(value);
+  }
+
+  const boost::optional<std::string>& GetCorrelationId() const {
+    return x_correlation_id;
+  }
+  void SetCorrelationId(std::string value) {
+    x_correlation_id = std::move(value);
+  }
+
+ private:
+  /// A subscription/Kafka Consumer is created in a specific server instance.
+  /// This base URL represents the specific server node on which this
+  /// subscription/Kafka Consumer was created. This base URL should be used
+  /// instead of the base URL returned by the `lookup` service when making
+  /// requests to these endpoints: /partitions, /offsets, /seek and /subscribe
+  /// (DELETE).
```

Comment:
Please remove Kafka from the comment.
Also remove ```(DELETE)``` as it is very strange and doesn't relate much to the C++ API.

Reply:
(DELETE) means the HTTP methods here.

We have two /subscribe endpoints:
1. For initiate subscription /subscribe (POST)
2. For delete subscription /subscribe (DELETE)

And it has a lot of sense, because we really need SubscribeResponse only for one version of /subscribe endpoint (DELETE).

Reply:
If you need it then you need to reference the StreamApi methods so that users of the SDK know what to do with this information. If I read your comment as is, my next question would be how can I access /seek or /partitions?

Reply:
This is private model, so users of the SDK has no access to it. We will use this model internally in StreamLayerClientImpl.
So only SDK developers can see these comments. And they should have no problems with access to /seek or /partitions.

Reply:
Except it is not /seek but StreamApi::Seek() and not /partitions but StreamApi::Partitions() or StreamApi::Consume() rather. So even for developers it is not so straight fwd to figure this out. Especially for those that do not have any experience with the StreamLayerClient or StreamApi.